app/controllers/main/routes.py

```diff
@@ -180,3 +182,20 @@ def searchUser(query):
 @main_bp.route('/contributors',methods = ['GET'])
 def contributors():
     return render_template("/contributors.html")
+
```

Comment:
Just making sure, is this section in the right place? Don't want it added here if it already exists somewhere else as a result of a merge moving code or something.

Reply:
I think is in the right place. I checked it and it does not exist anywhere apart from there